aggregator/src/aggregator/aggregation_job_init.rs

```diff
@@ -341,6 +341,21 @@ where
                             Ok(shares)
                         });
 
+                        // Reject reports after the task has expired. (§4.6.2.4, item 5)
+                        let shares = shares.and_then(|shares| {
+                            if let Some(task_end) = task.task_end() {
+                                if prepare_init
+                                    .report_share()
+                                    .metadata()
+                                    .time()
+                                    .is_after(task_end)
```

Comment:
The requirement says "Check if the report's timestamp is after the task's task_interval", and DAP time intervals are half-open, so I think we ought to treat reports as expired if their timestamp is equal to the task end time.